### PR TITLE
JOINDIN-642

### PR DESCRIPTION
**Pull request by exussum12**
Added reject button
Added exception as a use statement

### CODE REVIEW

File: app/src/Talk/TalkController.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 class TalkController extends BaseController
 {
-    protected function defineRoutes(\Slim\Slim $app)
```
Comment (exussum12) [inline app/src/Talk/TalkController.php]: imported on line 11.

**Comment by akrabat** [comment]
okay to test

**Comment by akrabat** [comment]
ok to test

**Comment by exussum12** [comment]
build now passes

**Comment by liam-wiltshire** [comment]
@exussum12 - Thanks for this, a feature we really needed. There is one minor UI change that'd be good if you get a chance - not a showstopper, hence I merged it anyway, but when you reject a talk claim, there is nothing in the UI that confirms this happens - if you look at approving a talk, the buttons get replaced with a 'Claim Approved' label - something similar for reject would be great if you get a minute :)

**Pull request merged**